100_Numpy_exercises_with_hints_with_solutions.md

```diff
@@ -72,8 +72,8 @@ print(Z)
 `hint: reshape`
 
 ```python
-nz = np.nonzero([1,2,0,0,4,0])
-print(nz)
+Z = np.arange(9).reshape((3, 3))
```

Comment:
Why this change ?

Reply:
The current solution corresponds to question 10.

Reply:
My bad... Thanks for fixing it!